src/types/markers.js

```diff
@@ -338,6 +338,13 @@ export type NetworkPayload = {|
   // happens on the process' main thread.
   endTime: Milliseconds,
 
+  // fetchStart doesn't exist directly in raw markers. This is added in the
+  // deriving process and represents the junction between START and END markers.
+  // This is the same value as the start marker's endTime and the end marker's
```

Comment:
I believe the startTime and endTime will be two different `mozilla::TimeStamp` calls, so there could be a tiny variation between the start marker's endTime, and the end marker's startTime. Perhaps it would be more accurate to describe this as the end marker's start time only.

Reply:
The gecko code keeps the actual value so we're positive that it will be the exact same value :)